### DISCUSSION TITLE
I want to know how the context work

### DISCUSSION THREAD

**Discussion by Frog863** [crewAIInc/crewAI#1085] [Q&A]
Sometimes the context of one action's prompt dosen't include the output of the previous task,but the answer also include the informetion.
Isn't the content I see in the 'action' something that is directly input to the LLM?

**Comment by zinyando** [comment]
Sorry, but your question is not very clear. You can explicitly define the context required for a task as shown in the docs here https://docs.crewai.com/core-concepts/Tasks/#referring-to-other-tasks

What are you trying to do and which results are you getting?
